### DISCUSSION TITLE
Questions on IgxDataChartComponent

### DISCUSSION THREAD

**Discussion by KaushikEDS** [IgniteUI/igniteui-angular#12114] [General]
Question 1: 
In a data chart when we have multiple yAxisComponent (numeric in my case) we seem to have multiple grid lines. 
The grid lines look messy, is there a way to synchronize the grid lines
![image](https://user-images.githubusercontent.com/79090369/190393334-b8f9fb16-7125-4c1a-a322-e788a3b46dc7.png)

Question 2: 
IgxCrosshairLayer component, how do we enable the annotation on the respective axis as we have in the category/financial chart?
Image of crosshair value on axis in financial chart
![image](https://user-images.githubusercontent.com/79090369/190394147-14f43e56-6281-439a-9b08-30b0a2206282.png)

**Comment by agoldenbaum** [comment]
Hello KaushikEDS,

I will answer your questions in the order they were asked.

**1.** I am under the impression that you are auto-generating the values of the NumericYAxis elements in your chart in this case. In this case, the axes may not be aligned as where the lines are drawn will be based on your data.

A couple of things that you can do in this case is to set the **minimumValue / maximumValue** properties of your NumericYAxis and then set the Interval property such that the number of labels and gridlines are synchronized, as this property determines the frequency that the labels and the major gridlines draw at.

Alternatively, if you do not wish to show the gridlines for a particular axis, you can set the **majorStroke** of the axis to “transparent.”

**2.** In order to show the annotation on the axis for the IgxCrosshairLayer, you need to set the **isAxisAnnotationEnabled** property on the layer to true. If it still does not show up, you are likely missing a module, as the IgxDataChartAnnotationModule is required for this behavior to work.

Please let me know if you have any other questions or concerns on this matter.

**Reply by KaushikEDS** [reply]
Thanks, @agoldenbaum that helps.
